openvdb/points/AttributeSet.h

```diff
@@ -540,6 +558,82 @@ class OPENVDB_API AttributeSet::Descriptor
 #endif
 }; // class Descriptor
 
+
+////////////////////////////////////////
+
+/// @brief A convenience class that collates the Descriptor with metadata that
+/// is only available on each AttributeArray, such as whether the array is
+/// uniform or not. This is useful for querying and during construction of new
+/// AttributeSets.
+/// @note For performance reasons, this class uses copy-on-write for storing
+/// the Descriptor so any modification of the source Descriptor also requires
+/// a deep-copy for the data in this class to remain valid.
```

Comment:
This comment doesn't make sense. Are you trying to say that if modify the attributes you need to rebuild an instance of this class?

Reply:
Yes exactly. I agree the wording can be improved here.